widgetssdk/src/main/java/com/glia/widgets/core/notification/NotificationActionReceiver.kt

```diff
@@ -8,6 +8,13 @@ import com.glia.widgets.di.Dependencies
 import com.glia.widgets.helper.Logger
 import com.glia.widgets.helper.TAG
 
+/**
+ * Glia internal class.
+ *
+ * Will be automatically added to integrator Manifest by Manifest merger during compilation.
+ *
+ * This Receiver is used to listen to push notification actions (e.g: button clicks)
```

Comment:
Activities and services are written with lowercase first letter, let's do the same with receiver, for consistency.
```suggestion
 * This receiver is used to listen to push notification actions (e.g: button clicks)
```